docker-compose/integrator-broker-analytics/docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
       retries: 10
       start_period: 15s
   wso2ei-analytics:
-    image: docker.wso2.com/wso2ei-analytics:6.1.1
+    image: wso2ei-analytics:6.2.0
```

Comment:
Why have we removed the docker.wso2.com prefix?

Reply:
@msmshariq will fix it. Couldn't revert the change after testing was finished.